libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -1261,7 +1259,8 @@ string ABIFunctions::abiDecodingFunctionCalldataStruct(StructType const& _type)
 		)"};
 		w("functionName", functionName);
 		w("readableTypeName", _type.toString(true));
-		w("minimumSize", to_string(_type.calldataEncodedSize(true)));
+		// TODO: This used padded calldatasize before. Any reason for that?
```

Comment:
Yes, structs are always padded.

Reply:
Yeah ok - as a matter of fact it'll be the same value anyways (the padding argument doesn't make a difference for structs)...

Generally I'm not sure, though, whether decoding should enforce the presence of padding at the very end of calldata - but we can discuss that separately, resp. check whether the behaviour of the decoder is consistent there. This is related to https://github.com/ethereum/solidity/issues/7180 and the question whether missing padding at the end of a single bytes argument should make the decoder throw...

But for the purpose of this PR, I'll change it to padded ``calldataEncodedSize`` anyways.

Reply:
See https://github.com/ethereum/solidity/issues/7200 now.